src/tools/ceph_osdomap_tool.cc

```diff
@@ -180,6 +180,12 @@ int main(int argc, char **argv) {
     }
     for (auto i : headers)
       std::cout << i << std::endl;
+  } else if (cmd == "corrupt") {
```

Comment:
it would be helpful to have a test that uses this, and checks that the repair op works on the corrupted store